dts/arm/silabs/efm32pg12b.dtsi

```diff
@@ -182,6 +182,22 @@
 				#gpio-cells = <2>;
 			};
 		};
+
+		wdog0: wdog@40052000 {
```

Comment:
Add an alias for 'watchdog-0' & `watchdog-1` in this file.

Reply:
Added 2 aliases